source/bao_hyp/config.rst

```diff
@@ -363,8 +363,9 @@ where:
 - ``size`` [mandatory] - corresponds to the size of the device memory region;
 
 .. note::
-  It is mandatory for ``base`` and ``size`` to align with the smallest page size of the architecture.
-  For MMU systems, this typically aligns to 4K, while for MPU systems, it aligns to 64 bytes.
+  It is mandatory for ``base`` and ``size`` to align with the smallest page size of the
+  architecture. For MMU systems, this typically aligns to 4K, while for MPU systems, it aligns to
+  64 bytes. Please note that for MPU systems there is no virtual address (VA).
```

Comment:
Currently, the VA address must be set anyway for MPU systems

Reply:
Thanks for clarifying, updated in https://github.com/bao-project/bao-docs/pull/78/commits/a139c992247e67ff35c787efe7afd279a15a7ff7!